pkg/virt-controller/ipamclaims/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
+  - sig-network-approvers
```

Comment:
@fabiand 
I added it per your request, only to give power but not liability for sig-network
can i remove it please ?
(also from reviewers)
sig-network should not need to maintain

Reply:
@phoracek 
Removed it
there is no any liability from sig-network to maintain it